feat(settings): Add support to configure fragment types via tables (#370)
twisted/towncrier: README.rst

    underlines = "=-~"
    wrap = false  # Wrap text to 79 characters
    all_bullets = true  # make all fragments bullet points

>     fragment_types = [] # A list of strings of strings with custom fragment types. Leave it empty if you want to use the default ones.

Reviewer: Is this still needed?

If a single file is used, the content of that file gets overwritten each time.

If ``title_format`` is unspecified or an empty string, the default format will be used.
If set to ``false``, no title will be created.
This can be useful if the specified template creates the title itself.

Furthermore, you can customize each of your own fragment types using:

.. code-block:: toml

    [tool.towncrier]

>     [tool.towncrier.feat]
>     [tool.towncrier.fix]

Reviewer: ```suggestion
    # To add custom fragment types, with default setting, just add an empty section.
    [tool.towncrier.feat]
    [tool.towncrier.fix]
```

>     [tool.towncrier.chore]
>         name = "Other Tasks"
>         showcontent = false

Reviewer: ```suggestion
    # Custom fragment types can have custom attributes
    # that are used when rendering the result based on the template.
    [tool.towncrier.chore]
        name = "Other Tasks"
        showcontent = false
```

    underlines = "=-~"
    wrap = false  # Wrap text to 79 characters
    all_bullets = true  # make all fragments bullet points

>     fragment_types = [] # A list of strings with custom fragment types. Leave it empty if you want to use the default ones.

Reviewer: I  think that due to lack the of the docs documentation, we ended up with documentation split between README and docs/

It would be nice to have fewer docs in the README.
README should just be a quick introduction to all the pointers for the project - docs, download, contributing... etc

```suggestion
    fragment_types = [] # A list strings to overwrite the default ones. If you want to reuse the default types you need to include them here. Leave it empty if you want to use the default ones.
```
